src/unleash-proxy.ts

```diff
@@ -44,11 +57,83 @@ export default class UnleashProxy {
         this.middleware = router;
 
         // Routes
-        router.get('/health', this.health.bind(this));
-        router.get('/', this.getEnabledToggles.bind(this));
-        router.post('/', this.lookupToggles.bind(this));
-        router.post('/client/metrics', this.registerMetrics.bind(this));
-        router.get('/client/features', this.unleashApi.bind(this));
+        router.get(
+            '/health',
+            openApiService.validPath({
+                responses: withStandardResponses(200, 503)(),
+            }),
+            this.health.bind(this),
+        );
+
+        router.get(
+            '/',
+            openApiService.validPath({
+                parameters: [
+                    ['appName', "Your application's name"],
+                    ['userId', "The current user's ID"],
+                    ['sessionId', "The current session's ID"],
+                    ['remoteAddress', "Your application's IP address"],
+                    ['properties', 'Additional properties'],
+                ].map(([k, v]) => ({
+                    name: k,
+                    description: v,
+                    in: 'query',
+                })),
+                responses: withStandardResponses(
+                    401,
+                    503,
+                )({ 200: featuresResponse }),
```

Comment:
Maybe we could simplify `withStandardResponses` a bit by limiting it to errors, e.g.:

```ts
openApiService.validPath({
    responses: {
        ...createErrorResponses(401, 503),
        200: featuresResponse
    }
});
```

Reply:
Interesting. Why would you want to limit it to errors?

Reply:
Just a readability thing. Since `withStandardResponses` doesn't do anything with the `200`, something like `createErrorResponses` might be easier to understand from the outside. Again really subjective, so up to you.

Reply:
I'm not sure I understand: the 200 in `withStandardResponses` is an empty 200 OK response. What do you mean by "doesn't do anything" with it?

Reply:
Sorry, tricky to explain. From the outside it looked like `withStandardResponses` might modify `featuresResponse` somehow. The alternative signature seems slightly simpler. But I missed the empty `200 case`, so it would be `createDefaultResponses` or something, you're right.

Reply:
You're completely right that spreading in an object seems to be the more JS and readable way to go here. Simplified in f62b1cbb0f88b70573da936b0c871a4244185bc9. Thanks! 🙏🏼